packages/help-center/src/hooks/index.ts

```diff
@@ -0,0 +1,4 @@
+export { useActiveSupportTicketsQuery } from './use-active-support-tickets-query';
```

Comment:
Is it ok to have one index file for the hooks?
It made sense for me..

Reply:
will remove it since we don't do it

Reply:
I suggest that we don't, to make tree shaking easier for Webpack and deleting unused code easier for us in the future.